extensions/tekton/generator/go.mod

```diff
@@ -0,0 +1,82 @@
+module github.com/fabric8io/kubernetes-client/extensions/tekton/generator
+
+require (
+	github.com/fabric8io/kubernetes-client/generator v0.0.0
+	github.com/tektoncd/pipeline v0.46.0
```

Comment:
I see that [0.47.0](https://github.com/tektoncd/pipeline/tree/v0.47.0) also got released a few days ago. Shall we bump to latest version?